### ISSUE TITLE
Feature need of datatable and picture in reply

### ISSUE THREAD

**Issue by littlepanda0716** [AI-Yash/st-chat#6]
Such a great component! I’m trying to use it to make a QA system, are you planning to add data table and picture in chat?

**Comment by yashppawar** [comment]
Will try to implement it ASAP if possible

**Comment by subhajit2001** [comment]
Yes it will be better if you implement the datatable and a procedure to change the picture of the user and the bot

**Comment by littlepanda0716** [comment]
hello, is there any chance to add this feature?
